cloudbank-v32/checks/pom.xml

```diff
@@ -13,9 +13,9 @@
     <version>0.0.1-SNAPSHOT</version>
     <name>checks</name>
     <description>Check Processing Application</description>
-    <properties>
+    <!-- <properties>
```

Comment:
did you mean to leave commented out code?

Reply:
Absolutely not :-)